File: Web/ClientApp/src/app/models/et/skillMatrixGroup.ts
import { EntityBase } from "../entityBase"
import { Guid } from "guid-typescript"
import { SkillMatrixSubject } from "./skillMatrixSubject";

export class SkillMatrixGroup extends EntityBase {
    groupId: Guid;
    groupName: string;
    description: string;
    active: boolean;
    skillMatrixSubjects: SkillMatrixSubject[]

    constructor() {
        super();
        this.skillMatrixSubjects = []
    }
}

File: Domain/Entities/ET/SkillMatrixGroup.cs
using System;
using System.Collections.Generic;


namespace Domain.Entities.ET;

public class SkillMatrixGroup : EntityBase
{
    public Guid? GroupId { get; set; }
    public string GroupName { get; set; }
    public string Description { get; set; }
    public bool Active { get; set; }
    public ICollection<SkillMatrixSubject> SkillMatrixSubjects { get; set; }
}
